.github/CODEOWNERS

```
@@ -3,8 +3,8 @@
 ###########
 
 # Catch all for whole project
-* @CIPop @ericwol-msft @ewertons @RLeclair @vaavva @zhkon
+* @ericwol-msft @ewertons @RLeclair @vaavva @zhkon
```

```suggestion
* @ericwol-msft @ewertons @RLeclair @vaavva @zhen9910
```
I think this is the right gh username for Zhen